### PR TITLE
Use Datastore for persistent comments

### PR DESCRIPTION
**Pull request by andres-holguin**
Begin use of Datastore by storing comments. Confirmed working by looking at the Datastore admin page. Have not yet implemented loading comments from Datastore.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -37,8 +41,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
     response.sendRedirect("/index.html");
   }
+
+  private void addNewComment(String commentText) {
+    comments.add(commentText); //TO-DO: Remove once comments can be loaded from Datastore
+    
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("text", commentText);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```
Comment (cftorres) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: Just thinking ahead-- you'll probably want to make this an instance variable so you can use the same instance to load comments from Datastore as well which it looks like you'll be doing next, based on your comments, and I assume will be in a different function.

**Comment by andres-holguin** [comment]
Implemented loading of comments from Datastore through a query.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (andres-holguin) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: That makes sense. This line is particularly long so it's nice to not have to rewrite it haha. I added that change in the last commit.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -20,25 +20,60 @@
...
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String newComment = request.getParameter("new-comment");
-    comments.add(newComment);
+    String commentText = request.getParameter("comment-text");
+    addNewComment(commentText);
     response.sendRedirect("/index.html");
   }
+
+  /**
+   * Helper function that returns an Array of strings with all the comments in Datastore
```
Comment (nickKelman) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: s/Array/ArrayList

**Pull request merged**